src/views/TaxFiling/taxFilingView.scss

```diff
@@ -0,0 +1,34 @@
+.Layer__tax-filing {
+  max-width: 1460px;
```

Comment:
This magic number is the same magic number used for `.Layer__view-header__content`.